pykg2tbl/service.py

```diff
@@ -82,16 +154,13 @@ def __init__(self, *files):
             )
 
     @staticmethod
-    def reslist_to_dict(reslist: list):
+    def query_result_to_dict(reslist: list):
         return [{str(v): str(row[v]) for v in reslist.vars} for row in reslist]
-        # TODO decide later on proper conversion to remove rdflib specifics and
-        #    create reusable data dict for conversion through query results
-        #   (pandas wrapper)
 
     def query(self, sparql: str) -> QueryResult:
         log.debug(f"executing sparql {sparql}")
         reslist = self.graph.query(sparql)
-        return QueryResult(KGFileSource.reslist_to_dict(reslist))
+        return QueryResult(self.query_result_to_dict(reslist))
```

Comment:
why not called as a static method?

Reply:
What is the pratical difference between this:
`        return QueryResult(self.query_result_to_dict(reslist))`
and this:
`        return QueryResult(KGFileSource.query_result_to_dict(reslist))`

?

For this case, I have been searching the web for an answer, but still confused on what should be the best practice here

Reply:
perfect, got it!